buildSrc/src/main/groovy/nva-orcid-client.java-common-conventions.gradle

```diff
@@ -16,11 +16,24 @@ group = 'com.github.bibsysdev'
 repositories {
     mavenCentral()
     maven { url "https://jitpack.io" }
+    maven { url "https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/release" }
 }
 
 dependencies {
 
     testImplementation libs.bundles.testing
+    testImplementation libs.mockito.core
+    testImplementation libs.hamcrest.base
+    testImplementation libs.hamcrest.core
+    testRuntimeOnly libs.junit.jupiter.engine
+    testImplementation libs.junit.jupiter.api
+    testImplementation libs.junit.vintage.engine
```

Comment:
Do we need this?

Reply:
copy paste from publication-api. 
I can try to remove those and see if it will compile now as I have fixed the gradle issues I had yesterday.

Reply:
Removed